streaming/src/main/scala/org/apache/spark/streaming/dstream/InputDStream.scala

```diff
@@ -49,6 +49,9 @@ abstract class InputDStream[T: ClassTag] (@transient ssc_ : StreamingContext)
    */
   private[streaming] def name: String = s"${getClass.getSimpleName}-$id"
 
+  /** Human-friendly scope name to use in place of generic operation names (e.g. createStream). */
+  protected override val customScopeName: Option[String] = Some(s"input stream [$id]")
```

Comment:
because this is only used in `DStream` and overridden in its children. Classes outside of `DStream` don't need to access it.